### DISCUSSION TITLE
YAML configuration is deprecated

### DISCUSSION THREAD

**Discussion by jo8be** [alandtse/alexa_media_player#2706] [Q&A]
I was about to update to Alexa Media Player 5.0, but then read: 
_YAML configuration is deprecated. Users must manually remove alexa_media from configuration.yaml file and add the integration via UI to avoid warning messages and future failure._

That sounds threatening ... and I can't figure out what that means for my setup. I certainly did some copy/paste in the yaml. 

Is there a more detailed explanation/documentation of that change so I can estimate, how likely it will break my setup and what exactly to do then. 
I'm not so experienced with HA, as I touch it every 10 weeks or so to add or change something.

Update: 
I think I confused configuration.yaml with automations.yaml. I have no mention of the media player in configuration.yaml, so I guess the update will cause no problems.  
Update to 5.0 went fine, no problems (so far).

**Comment by danielbrunt57** [comment]
While YAML config is now officially deprecated, 5.0.0 doesn't prevent existing YAML setups from working but does issue a warning advising you to act now to avoid future failure.  You should heed the warning and remove `alexa_media:` from `configuration.yaml`, restart and add the integration via UI Integrations.

**Reply by rodrigofragadf** [reply]
> While YAML config is now officially deprecated, 5.0.0 doesn't prevent existing YAML setups from working but does issue a warning advising you to act now to avoid future failure. You should heed the warning and remove `alexa_media:` from `configuration.yaml`, restart and add the integration via UI Integrations.

I updated to version 5 and removed alexa_media: from configuration.yaml, restarted HA, but the integration was still there. If I remove alexa_media: from the configuration file, the integration stops working. What should I do? I also couldn't find where to edit the filters. Can you tell me how to configure them in this new version?

**Reply by danielbrunt57** [reply]
I'm somewhat confused by your question.  
> and removed alexa_media: from configuration.yaml, restarted HA, but the integration was still there.

Then...
> If I remove alexa_media: from the configuration file, the integration stops working.

If you remove `alexa_media` from `configuration.yaml` and restart HA, the integration should not be working nor should it exist. 
You then need to go to Settings > Devices & services (Integrations) and click on `+ ADD INTEGRATION`.
After it's installed & working if you want to change the filters, click the integration in the UI and when it opens, select `CONFIGURE`.

On the other hand, it might be possible that after removing `alexa_media` from `configuration.yaml`, the entry it created in `core.config_entries` might still exist, in which case I'd expect to see it still listed as an already installed integration in the UI...
